core/src/main/java/io/questdb/cairo/sql/Function.java

```diff
@@ -203,6 +203,10 @@ default boolean isConstant() {
         return false;
     }
 
+    default boolean isNonDeterministic() {
```

Comment:
Should be true for:

- TodayWithTimezoneFunctionFactory, all other today* functions
- TomorrowFunctionFactory, all other tomorrow* functions
- YesterdayFunctionFactory, and yesterday* functions
- Bind functions
- BinaryFunction, MultiArgFunction, TernaryFunction, UnaryFunction should return true if one of their argument return true
- TimestampSequenceFunction

Reply:
Fixed in b9bd6bd01e533b99518c63d19009242cae6a2fc7

> * TodayWithTimezoneFunctionFactory, all other today* functions
> * TomorrowFunctionFactory, all other tomorrow* functions
> * YesterdayFunctionFactory, and yesterday* functions

These are already marked correctly thanks to `AbstractDayIntervalFunction`